tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+"""
+   Copyright 2020 Google LLC
```

Comment:
nit: make license header `#` comment not multiline string.
Currently license is you module level docstring which is not helpful for IDEs / docs tools parsing your module.